utopian-io/v2.utopian.io
# Bugfix/api wrapper

### Verify that

### My PR doesn't concern a Jira task
There cases where the API wrapper crashes because of bad error handling.
I've added a way to display errors received from the API to the user. It shouldn't occur often because the UI controls should prevent the user from sending bad data.
The action can be used in other cases to display the error to the user.

---

### **Test plan**

_Display an error notification_

* I don't have an immediate error to display so the easiest way is to throw an error by modifying the code
* Log in, go to the create project page using the button at the bottom of the homepage
* Add `  throw Boom.badData('document-does-not-exist')` at the start of `getFeaturedProjects` and wait for the API to reload
* Click on the logo to be redirected to the homepage

**Expected result:** You should see a notification error in the bottom right
